west.yml

```diff
@@ -219,7 +219,7 @@ manifest:
         - hal
     - name: hal_rpi_pico
       path: modules/hal/rpi_pico
-      revision: fba7162cc7bee06d0149622bbcaac4e41062d368
+      revision: pull/6/head
```

Comment:
How can I help get this PR approved and merged?

Reply:
let's get https://github.com/zephyrproject-rtos/zephyr/pull/76986 in then you can rebase on top of it and drop the duplicate commits